02/app.js

```diff
@@ -1,7 +1,75 @@
 
 /* rozwiązanie z pętlą for */
-const x = 4;
+// const x = 4;
+// const x = getNumber('Podaj liczbę z zakresu 0 - 9');
+
+// function getNumber(numberFromUser){
+//     const x = prompt(numberFromUser)
+//         if(checkNumber(x)){
+//             return x;
+//         }
+// }
+
+// function checkNumber(numb){
+//     if(numb >= 1 && numb <=9){
+//         return true;
+//     }
+//     else {
+//        alert('Podano zla liczbe! Podaj liczbe z zakresu 1 - 9');
+//        getNumber();
+//     }
+// }
+
+// for(i=1;i<=9;i++){
+//     let result=0;
+//     result = i * x;
+//     console.log(i + ' x ' + x + ' = ' + result);
+//  }
+
+
+/* rozwiązanie z pętlą while  */
+// const a = 3;
+// const n = 4;
+
+const a = getNumberPow('Podaj liczbę z zakresu 0 - 9 - podstawa potęgi');
```

Comment:
Zwróć uwagę, że podajesz 1 argument, a Twoja funkcja ma zdefiniowane dwa parametry!
Dodatkowo ten parametr to `numberFrmUserA`, który w moim odczuciu oznacza "liczba pierwsz", a tak na prawdę jest to wiadomość. Poprawnie funkcja ta powinna wyglądać tak:
```
function getNumberPow(message){
    const num = prompt(message)
    if(checkNumberPow(num)){
        return num;
    }
        
    return null;
}
```
Zwróć uwagę, że tutaj wystarcza mo operacja na jednym elmencie bo ta funkcja obiera od użytknika dane i je zwraca. Mogę ją wwołąć wieloktronie i nie muszę tutaj powtarzać wielu liczb